cnd/src/config/file.rs

```diff
@@ -237,4 +268,104 @@ node_url = "http://example.com/"
 
         assert_eq!(file, file_with_effective_settings)
     }
+
+    #[test]
+    fn bitcoin_deserializes_correctly() {
+        let file_contents = vec![
+            r#"
+            network = "mainnet"
+            [bitcoind]
+            node_url = "http://example.com:8332"
+            "#,
+            r#"
+            network = "testnet"
+            [bitcoind]
+            node_url = "http://example.com:18332"
+            "#,
+            r#"
+            network = "regtest"
+            [bitcoind]
+            node_url = "http://example.com:18443"
+            "#,
+        ];
+
+        let expected = vec![
+            Bitcoin {
+                network: bitcoin::Network::Bitcoin,
+                bitcoind: Some(Bitcoind {
+                    node_url: Url::parse("http://example.com:8332").unwrap(),
+                }),
+            },
+            Bitcoin {
+                network: bitcoin::Network::Testnet,
+                bitcoind: Some(Bitcoind {
+                    node_url: Url::parse("http://example.com:18332").unwrap(),
+                }),
+            },
+            Bitcoin {
+                network: bitcoin::Network::Regtest,
+                bitcoind: Some(Bitcoind {
+                    node_url: Url::parse("http://example.com:18443").unwrap(),
```

Comment:
Is there any relevance to these port numbers?  Are they different for a reason?  Not super important though.

Reply:
Oh, these are the default Bitcoin ports per network.

Reply:
Yes, those are the defaults used by `bitcoind`.